config/crontab-example

```diff
@@ -35,6 +35,8 @@ MAILTO=!!(*= $mailto *)!!
 # Once a day on all servers
 43 2 * * * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/request-creation-graph
 48 2 * * * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/user-use-graph
+54 2 * * * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/cleanup-holding-pen
```

Comment:
Once a day seems a bit much; almost not worth showing a history at that point. Maybe once a week?